devdoc/sm_requirements.md

```diff
@@ -92,6 +93,8 @@ typedef struct SM_HANDLE_DATA_TAG* SM_HANDLE;
 
 MU_DEFINE_ENUM(SM_RESULT, SM_RESULT_VALUES);
 
+typedef void(*SM_OPENED_DRAINING_TO_CLOSE_COMPLETE_CALLBACK)(void* context);
```

Comment:
>SM_OPENED_DRAINING_TO_CLOSE_COMPLETE_CALLBACK [](http://example.com/codeflow?start=14&length=45)

I will have to say that "DRAINING TO CLOSE" is very rather internal name and never was exposed to the user before.

So as far as discoverability of the API goes, there can be better names. 

How about considering renaming it to "ON_SM_CLOSING" or similar.